### PR TITLE
CCX0022 Integration 2022-03-15

### PR DESCRIPTION
**Pull request by vgoodric**
Fix https://github.com/adobe/express-website-issues/issues/332

Test URL: https://ccx0022--express-website--adobe.hlx.page/express/create/flyer?lighthouse=on

### CODE REVIEW

**Comment by rofe** [comment]
@vgoodric could you please fix the linting issues and add a test URL to the PR description?

**Comment by vgoodric** [comment]
@rofe, @davidnuescheler  One of the pages with sixcols is this: https://www.adobe.com/express/create/flyer but I'm not sure if you mean the localhost address.  Truth is I still can't get the local installation to work.  So I can't test my code.  But I did just push an update where I moved the code to the template-list

**Comment by davidnuescheler** [comment]
a test URL would be a helix URL that point to the branch, so we can all look at the code and see if it does what it is supposed to do... i this case it would be: https://ccx0022--express-website--adobe.hlx.page/express/create/flyer if that's page where we should see the changes....

**Comment by vgoodric** [comment]
Eventual production url for QA: https://www.adobe.com/express/create/flyer
Test URL: https://ccx0022--express-website--adobe.hlx.page/express/create/flyer

**Review by davidnuescheler — APPROVED** [review]

**Review by rofe — APPROVED** [review]

**Comment by rofe** [comment]
@vgoodric great job! The only thing missing to get this PR to green is you signing the [Adobe CLS](https://github.com/adobe/express-website/pull/372/checks?check_run_id=5603899066)

**Comment by vgoodric** [comment]
@rofe ok, I've signed it

**Comment by rofe** [comment]
@timliu-adobe this is ready to be pushed live. Is there a specific timing for the deployment, or asap?

File: express/blocks/template-list/template-list.css [hunk 1]
```diff
@@ -356,6 +356,17 @@ main .template-list.sixcols .template.placeholder {
   margin: 15px auto;
 }
 
+/* template-list-scrollbutton */
+main .template-list.sixcols .template-list-scrollbutton {
+  padding: 7px 8px;
+  position: fixed;
+  bottom: 0;
+  right: 0;
+  z-index: 1;
```
Comment (rofe) [inline express/blocks/template-list/template-list.css:365]:
```suggestion
  z-index: 3;
```
Increase z-index to make sure scroll button stays above floating CTA button.

**Review by rofe — CHANGES_REQUESTED** [review]

**Comment by timliu-adobe** [comment]
hey @rofe @vgoodric—QA is complete for this, and we are good to push it live!

**Pull request merged**